pkg/felt/other.go

```diff
@@ -0,0 +1,150 @@
+package felt
+
+import (
+	"github.com/NethermindEth/juno/pkg/common"
+)
+
+var (
+	Felt0 = new(Felt).SetZero()
+	Felt1 = new(Felt).SetOne()
+	Felt2 = new(Felt).SetUint64(2)
+	Felt3 = new(Felt).SetUint64(3)
+)
+
```

Comment:
I think the burden of these should be on the client. It doesn't make sense to impose this cost on everyone that wants to use the package especially when we have convenience methods defined that archive this without the significant memory overhead.